.github/workflows/test.yml

```
@@ -50,15 +89,16 @@ jobs:
           mkdir qemu-data
           touch qemu-data/hdd.qcow2
           ./qemu-run.sh nographic os &
+
       - name: Create directory for logs
         shell: bash
-        id: log_dirs
+        id: create_log_dirs
         run: |
-          timestamp=$(date -u +%Y-%m-%dT%H:%M:%S%Z)
-          directory="/tmp/dts-test-ci-${timestamp}"
-          mkdir $directory
-          echo "directory=$directory" >> "$GITHUB_OUTPUT"
-      - name: Run tests
+          export LOG_DIR="/tmp/dst-test-ci-${{ github.run_id }}/${{ matrix.include.test }}"
```

```suggestion
          LOG_DIR="/tmp/dst-test-ci-${{ github.run_id }}/${{ matrix.include.test }}"
```

`export` won't work as those command are run as a script (workflow creates temporary file with those commands and passes it to `sh`/`bash`

---

8ae4197eb4fb062a504f52ada2b282b47c06b852